src/blueprints/playlist.py

```
@@ -86,6 +86,24 @@ def _windows_overlap(start_a: int, end_a: int, start_b: int, end_b: int) -> bool
     return False
 
 
+def _default_overlap_warning(start_min, end_min, playlists):
```

Ruff (0.15.9)

[warning] Missing return type annotation for private function `_default_overlap_warning`

Add return type annotation: `str | None`

(ANN202)